Dockerfile.staticbuild

```diff
@@ -52,7 +52,7 @@ RUN set -x && \
     tar -xvf icu4c-62_1-src.tgz && \
     cd icu/source && \
     ./configure --with-data-packaging=static --enable-static --enable-shared && \
-    make -j && make install
+    make -j 1 && make install
```

Comment:
Please, use `make -j $(nproc)` instead. It is likely fast enough and likely will not eat too much memory.

Reply:
Sure, changed.